src/test/resources/com/ibm/vCard/vcard_escaping.txt

```diff
@@ -16,5 +16,5 @@ SOUND;VALUE=URL:file///multimed/audio/jqpublic.wav
 UID:19950401-080045-40000F192713-0052
 REV:20080424T195243Z
 NOTE;ENCODING=QUOTED-PRINTABLE:Testing escaping- : should appear & /; comma should appear & /
- (CRLF) should appear & // should appear, all in the same field 
-END:VCARD
\ No newline at end of file
+ (LF) should appear & // should appear, all in the same field 
```

Comment:
Revert text, and line endings, to CRLF in this test data. 

We may have to change some git settings to make it not LF-ify the line endings by default. vCard, like many IETF standards, is fundamentally CRLF stuff.